src/components/Favorite/Favorite.module.scss

```diff
@@ -5,15 +5,29 @@
   margin: 0 auto;
   flex-shrink: 0;
   width: calc(33% - 12px);
-  background: $color-light;
+  //background: white;
   padding: ($base-size * 1.5);
   border-radius: $radius;
-  box-shadow: 0 math.div($base-size, 4) 0 rgba(0,0,0,0.3);
+  //box-shadow: 0 math.div($base-size, 4) 0 rgba(0,0,0,0.3);
   color: $color-dark;
+  z-index: 3;
+
+
 }
 
 .cards {
   padding: math.div($base-size, 2) 0;
+  z-index: 2;
+
+
+}
+
+.nofav {
+  text-align: center;
+  margin-bottom: -55px;
```

Comment:
A czemu tak? Dosyć konkretny ten margines. Sprawdź jak będzie to wyglądało na mobile

Reply:
Na mobile nie ma problemu. Niczego nie zjada i nic nie ucina, nawet przy zmianie rozmiaru wyświetlanego okna.

Reply:
Ok, w najnowszym commicie użyłem warunku lub (alternatywnie) przekierowanie do home.